nighthawk/source/common/statistic_impl.cc

```diff
@@ -0,0 +1,212 @@
+#include "nighthawk/source/common/statistic_impl.h"
+
+#include <cmath>
+#include <stdio.h>
+
+#include "common/common/assert.h"
+
+namespace Nighthawk {
+
+void StatisticImpl::dumpToStdOut(const std::string& header) const {
+  ENVOY_LOG(info, "{}", header);
+  ENVOY_LOG(info, "#Completed: {}. Mean: {:.{}f}μs. Stdev: {:.{}f}μs.", count(), mean() / 1000, 2,
+            stdev() / 1000, 2);
+}
+
+void StatisticImpl::toProtoOutput(nighthawk::client::Output& output) {
+  output.set_request_count(count());
+  output.mutable_mean()->set_nanos(mean());
+  output.mutable_stdev()->set_nanos(stdev());
```

Comment:
No percentiles?

Reply:
`StatisticImpl` is a base class for the other implementations. `HdrStatistic` is currently the only one that provides percentiles and overrides this to do so.